SB2018/2018-01-12.md

```diff
@@ -0,0 +1,34 @@
+---
+to: All teams
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+Some important things to announce so keep reading:
+
+## Mentoring
+
+We're going to try and provide mentoring support for your team!
+
+If you're interested in having a mentor come and help you, **reply to this email with the times your teams meet**, and we'll see if we can assign someone to you!
+
+## Tech Days
+
+We're happy to announce some tech days!
+
+They'll be at the University of Southampton Electronics and Computer Science department in the £4 million electronics labs.
+
+Tech days start at 10:00am and finish at 5:00pm, Tech days times where you can come to us for robotics advice or help with technical issues. You also get to see how other teams are doing. We typically give a tour of the university facilities during this time.
```

Comment:
I think the first comma here should be a full stop. There's also no need to repeat the words "Tech days" quite so much

The next stanza is definitely missing a word; I think it's meant to be an "are". It also reads a bit oddly to me

How about:
> Tech days start at 10:00am and finish at 5:00pm. They are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. You also get to see how other teams are doing. We typically give a tour of the university facilities as well.